phylokit/core.py

```diff
@@ -4,9 +4,12 @@
 import numba
 import xarray
 
+import phylokit as pk
```

Comment:
This is because we are having the `@core.numba_njit` decorator in `core.py` which will cause circular import. 

Maybe we should move this decorator to somewhere else?

Reply:
> I guess that's an argument for only putting the minimum in `core.py` all right. Mabye move non-decorator stuff out of there?

I mean we define the decorator in `core.py` like this.

```python
def numba_njit(func, **kwargs):
    if ENABLE_NUMBA:  # pragma: no cover
        return numba.jit(func, **{**DEFAULT_NUMBA_ARGS, **kwargs})
    else:
        return func
```

Maybe we could move this function into a separate file I guess?

Reply:
We can leave it if it's too awkward and make an issue to track if you prefer?

Reply:
Agreed, haven't got a way to do it properly